### ISSUE TITLE
Display analytic rank for ECNF in more cases.

### ISSUE THREAD

**Issue by roed314** [LMFDB/lmfdb#6296]
When I look at [2.0.10691.1-1.0.1-a1](https://beta.lmfdb.org/L/4/10691e2/1.1/c1e2/0/0) the analytic rank is shown as "not available" but the [L-function](https://beta.lmfdb.org/L/4/10691e2/1.1/c1e2/0/0) shows the analytic rank as 1.  It would be nice to either port analytic ranks over to `db.ec_nfcurves` from L-functions or to update the ecnf pages on the fly when the information is available in the L-function.

**Comment by roed314** [comment]
See [this comment](https://github.com/LMFDB/lmfdb/pull/6290#issuecomment-2528569498) for some context.

**Comment by JohnCremona** [comment]
Another issue with this example is that the EC link on the L page is wrong, as it labels the conductor as 1.0.1 and not 1.1.  We fixed these old-style labels not that long ago, but the L-function pages are still using the old ones.

**Comment by JohnCremona** [comment]
Anyway, the curve's page https://www.lmfdb.org/EllipticCurve/2.0.10691.1/1.1/a/1 does show the analytic rank as 1, it is only the rank which is not known (upper bound 1 but we have no point of infinite order).

Close this?

**Comment by roed314** [comment]
Yes, this should be closed; I'm not sure if something has changed since December, or if I was just confused about the rank being shown as not available and thinking it was the analytic rank.

I'll create a separate issue about the broken link on the L page.
